docs/openapi.yml

```diff
@@ -1899,6 +1899,11 @@ components:
           description: regex of paths to be ignored for changes
           type: array
           example: [ "/target", "/.m2", "*/.git/*" ]
+        refPaths:
+          description: additional referenced paths to monitor
```

Comment:
Should we mention that referenced paths are for files that reside outside of the project folder?